Python/preprocessing/readme.md

```diff
@@ -85,29 +134,29 @@ Pre-built *.pyd with following environment:
 If you need other versions, please get more information from the [official FlowNet2-PyTorch website](https://github.com/NVIDIA/flownet2-pytorch).
 
 
-## 1.6. FFmpeg
+## 2.6. FFmpeg
 
 The pre-processing script is dependent on the Python interface of FFmpeg (`ffmpeg-python`), which depends on the `ffmpeg.exe` and `ffprob.exe` files.
 
 Please download the FFmpeg windows-x64-static binary package from the [official FFmpeg webside](https://www.ffmpeg.org/download.html).
 And add the root folder of ffmpeg.exe to the `PATH` environment variables of Windows.
 
 
-## 1.7. OmniPhotos
+## 2.7. OmniPhotos
 
 This dataset preprocessing script will call the OmniPhotos `Preprocessing` program to generate the sphere fitting mesh, etc.  
 Please download the OmniPhotos binaries or build OmniPhotos from source, and replace the absolute path of `Preprocessing.exe` in **config_omniphotos.yaml** with yours.
 
 
-# 2. Dataset preprocessing
+# 3. Dataset preprocessing
 
 There are two kinds of preprocessing:
 1. The input video captures contains multiple circles/swings, so we need additional steps to select the best circle motion, as described in [Section 2.2](#22-selecting-the-best-circle-openvslam-only).
```

Comment:
Please note that references to sections need to be updated after renumbering sections.